src/components/Timer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { displayMilliseconds } from '../utils/index';
+
+export class Timer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            time: new Date().getTime(),
+        };
+    }
+    componentDidMount() {
+        this.timerInterval = setInterval(
+            () => this.tick(),
+            1000,
+        );
+    }
+    componentWillUnmount() {
+        clearInterval(this.timerInterval);
+    }
+    tick() {
+        this.setState({
+            time: new Date().getTime(),
```

Comment:
@nazaninreihani should it be server-time? what happens if the client's machine datatime is not accurate?